### PR TITLE
#166790042: Redux setup

### PR DESCRIPTION
**Pull request by kagabof**
#### What does this PR do?

- Add redux setup

#### Description of Task to be completed?

- Add component folder
- Add homePage component for test
- Add redux action, reducers and configure store.

#### How should this be manually tested?

- By using redux in component

#### Any background context you want to provide?

- NONE

#### What are the relevant pivotal tracker stories?

[#166790042](https://www.pivotaltracker.com/story/show/166790042)

#### Screenshots (if appropriate)

- NONE

#### Questions:

- NONE

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+const App = () => (
+    <div>
+        <Switch>
+            <Route exact path="/" component={HomePage} />
+        </Switch>
+    </div>
+)
+
+export default App;
```
Comment (shaluchandwani) [inline src/components/App.js]: I have these components in my PR. They will not be required once it is merged.

File: src/components/homePage.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const HomePage = () => (
+    <div>
+        <h1>Hello Jawans , We are up and ready to go!!!</h1>
+    </div>
+);
+
+export default HomePage;
```
Comment (shaluchandwani) [inline src/components/homePage.js]: I have this component in my PR. They will not be required once it is merged.

**Review by mnzube — APPROVED** [review]
Nice

File: src/index.html [hunk 3]
```diff
@@ -1,14 +1,16 @@
...
+    <title>Authors Haven Jawans</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+
+<body>
+    <section id="app"></section>
+</body>
+
+</html>
```
Comment (shaluchandwani) [inline src/index.html]: Please add a line at the end of all the files. It's easier if you download the Prettier extension.

**Review by Joe-Joseph — COMMENTED** [review]
Apply those comments Shalu gave you, Then it will be good to go

**Review by pngabo — APPROVED** [review]

File: src/components/App.js (hunk 1 above)
Comment (kagabof) [reply src/components/App.js]: Yes, I rebase now yours are the one included.

**Review by shaluchandwani — APPROVED** [review]

**Review by Joe-Joseph — APPROVED** [review]

**Comment by billkabanga** [comment]
Good work on the task @kagabof, We shall also need to have a folder for `middlewares` or `thunks` under the redux folder. You also need to add a folder for containers under the `src` folder

**Pull request merged**